### ISSUE TITLE
$0 Funding easter egg

### ISSUE THREAD

**Issue by cedricwaxwing** [dOrgTech/landing-page-v3#68]
[![0 funding](https://user-images.githubusercontent.com/80348806/194379975-73394df3-2ef0-4703-9204-67e67489c601.png)](https://discord.com/channels/544647726982365194/1027236165520199802/1027238501646880829)

**Comment by orishim** [comment]
I agree with the need for an easter egg. someone I sent the site to asked me if this being at 0 was a bug, so an easter egg would make it clearer what we're saying here
